packages/website/src/icons/copy.svg

```diff
@@ -5,5 +5,5 @@
   height="18"
   fill="currentColor"
 >
-  <path d="M3 3v14h2V5h12V3H3zm4 4v14h14V7H7z"/>
+  <path d="M19,21H8V7H19M19,5H8A2,2 0 0,0 6,7V21A2,2 0 0,0 8,23H19A2,2 0 0,0 21,21V7A2,2 0 0,0 19,5M16,1H4A2,2 0 0,0 2,3V17H4V3H16V1Z" />
```

Comment:
@Josh-Cena do you think that we could expose those icons in docusaurus, if so i can make a PR for this there